apis/nucleus/src/components/Cell.jsx

```diff
@@ -383,18 +384,21 @@ const Cell = forwardRef(
 
     useEffect(() => {
       eventmixin(focusHandler.current);
+      focusHandler.current.blurCallback = (resetFocus) => {
+        if (focusHandler.current.onBlurHandler) {
+          focusHandler.current.onBlurHandler();
```

Comment:
need to pass on resetFocus to onBlurHandler (and maybe what cell it is connected to)